docs/introduction/developer-quickstart.md

```diff
@@ -6,22 +6,24 @@ sidebar_position: 4
 
 Obscuro executes bytecode in the EVM no different than Ethereum (or Arbitrum), so as a developer you can use all your existing code and tooling.
 
-The difference from Ethereum (and virtually all chains) is that private variables and the internal state of the contract while 
-executing are hidden from everyone (which includes node operators and the sequencer).
+The difference from Ethereum (and virtually all chains) is that private variables and the internal state of the contract while executing are hidden from everyone (which includes node operators and the sequencer).
```

Comment:
Rewrite as below to make it sound more positive. 

The main difference and advantage of Obscuro is that on Obscuro, during execution, private variables and the internal state of the contract are hidden from everyone, including node operators and the sequencer.

Reply:
Fixed